src/mpid/ch4/src/ch4_spawn.c

```diff
@@ -81,7 +94,7 @@ int MPID_Comm_spawn_multiple(int count, char *commands[], char **argvs[], const
         MPIR_ERR_CHECK(mpi_errno);
 
         /* FIXME: Are we only checking pmi_errcodes[0] and ignoring the rest? */
-        for (i = 0; i < total_num_processes; i++) {
+        for (int i = 0; i < total_num_processes; i++) {
             errcodes[i] = pmi_errcodes[0];
```

Comment:
not from this PR, but should this be?
```suggestion
            errcodes[i] = pmi_errcodes[i];
```

Reply:
The "FIXME" was also from me last time I looked at this code. I agree with you. Let me change it.